### DISCUSSION TITLE
alist挂载本地磁盘后多出了两个莫名其妙的文件夹

### DISCUSSION THREAD

**Discussion by Faxmell** [AlistGo/alist#3244] [Q&A]
![image](https://user-images.githubusercontent.com/108520655/216593983-3452e18d-b60c-4df3-bd64-2df6bd2394a2.png)
如图，我的e盘并没有这两个文件夹，也无法删除，有什么解决办法吗？

**Comment by anwen-anyi** [comment]
把你自己电脑的  隐藏文件/文件夹 打开看看

**Comment by drw2005** [comment]
那是你电脑里的两个隐藏文件夹，不想看到它，就到Alist后台把它隐藏掉就行了。
